features/networking/multus.feature

```diff
@@ -416,4 +416,248 @@ Feature: Multus-CNI related scenarios
     Then the output should contain "net1"
     And the output should contain "net2"
     And the output should contain 2 times:
-      | macvlan mode bridge |
+	    | macvlan mode bridge |
+
+ 
+  # @author anusaxen@redhat.com
+  # @case_id OCP-24490
+  @admin
+  Scenario: Pods can communicate each other with same vlan tag
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+    And I store all worker nodes to the :nodes clipboard
+    Given I have a project
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/NetworkAttachmentDefinitions/bridge-host-local-vlan.yaml |
+      | n | <%= project.name %>                                                                                                                               |
+    Then the step should succeed 
+    #Labeling a worker node to make sure couple of future pods to be scheduled on this node only
+    When I run the :label admin command with:
```

Comment:
@anuragthehatter  Thanks for the job. 
How about store the node name and using `nodeName` to make the pod to schedule the node you wanted. we better avoid updating the default label of worker.

Reply:
@zhaozhanqi Yep, i made sure about this. This step is not overwriting default label but just appending this new label to default. During cleanup just the appended label will be cleaned up and default one's will stay same

Reply:
btw could you let original commenter to mark conversations as resolved? If you mark, then it is hard for commenter to figure out which changes are to be reviewed.